src/main/java/org/rulez/demokracia/pdengine/WebSecurityConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.rulez.demokracia.pdengine;
+
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+	@Override
+	protected void configure(final HttpSecurity http) throws Exception {
+		http.cors().and().csrf().disable();
```

Comment:
Does it mean that we disable CORS and CSRF protection?
Why?

Reply:
Since we don't have authentication yet, all the REST requests return 403 if CORS and CSRF are enabled.

Reply:
I am thinking about ways to make sure this will not end up this way forever.
Maybe a FIXME comment is the way to go?